src/QtAVPlayer/qavplayer.cpp

```diff
@@ -426,7 +434,6 @@ void QAVPlayer::play()
         return;
     }
 
-    QMutexLocker locker(&d->waitMutex);
```

Comment:
seems the locker should be here?

Reply:
Haha, maybe I misunderstood you, but your previous comment was that this is not necessary :). After some thinking I convinced myself this is not necessary, because we are inside 'play', which means everything should be stopped. But sure, I can add locker back.